src/Spd.Resource.Repository/BizLicApplication/BizLicApplicationRepository.cs

```diff
@@ -322,12 +328,20 @@ private async Task SetOwner(spd_application app, Guid ownerId, CancellationToken
 
     private contact GetContact(Guid contactId)
     {
-        contact? contact = _context.contacts
-            .Where(c => c.contactid == contactId)
-            .Where(c => c.statecode == DynamicsConstants.StateCode_Active)
-            .FirstOrDefault();
+        contact? contact;
 
-        if (contact == null) throw new ArgumentException($"cannot find the contact with contactId : {contactId}");
+        try
```

Comment:
this GetContact function can be removed.
Use DynamicsContextLookupHelpers,GetContactById

Reply:
Fixed